README.md

```diff
@@ -20,9 +20,12 @@ A template is set up in the `Oneshot/` directory. See [`Oneshot/README.md`](Ones
 Install dependencies if needed:
 
 - debian/ubuntu: `sudo apt install cmake jq gmp gmp-devel`
-  (possibly `libgmp3-dev` instead of `gmp` and `gmp-devel`?)
+  (possibly `sudo apt install cmake jq libgmp3-dev libgmp3-dev` instead)
```

Comment:
```suggestion
  (possibly `sudo apt install cmake jq libgmp3-dev` instead)
```

Reply:
On my computer, it worked with `libgmp3-dev` twice. Is it redundant?

Reply:
Yes (at least I would be highly surprised if it is not redundant).

Reply:
Oh, they are not positional parameters! They are a list of things that should be installed, right?